odoo/osv/expression.py

```diff
@@ -441,6 +449,11 @@ def __init__(self, domain, model, alias=None, query=None):
         # parse the domain expression
         self.parse()
 
+    def _unaccent(self, field, _id=lambda x: x):
```

Comment:
The method is really nice, but the parameter `_id` in the API really feels awkward, since it's actually not a real parameter.

Reply:
Yeah that was a caching micro-optimisation as this is inside the compilation of `expression`, I think I did that before I checked how much local functions actually cost (bupkis).